packages/regl-worldview/src/WorldviewContext.js

```diff
@@ -222,12 +223,12 @@ export class WorldviewContext {
 
   _debouncedPaint = debounce(this.paint, 10);
 
-  readHitmap(
+  readHitmap = queuePromise((
```

Comment:
What do you think about dropping some early promises if the queued promises get too large or if the waiting time is too long? Could get a lot of queued promises if the user is rapidly clicking stacked objects in a changing scene.

Reply:
We can't drop early promises because every `readHitmap` function has to resolve, and they could have different arguments. However, what I could add here is a cache of the `readHitmap` result that is busted every time there are new arguments or commands updates.